packages/twenty-server/src/engine/core-modules/billing/billing.controller.ts

```diff
@@ -61,16 +65,38 @@ export class BillingController {
         return;
       }
 
-      await this.billingWehbookService.processStripeEvent(
+      await this.billingWebhookSubscriptionService.processStripeSubscriptionEvent(
         workspaceId,
         event.data,
       );
+      const customer = await this.stripeService.getCustomer(
+        event.data.object.customer as string,
```

Comment:
why do we need to cast as?

Reply:
because the  _event.data.object.customer_ is a  _Stripe.Subscription.customer_ which can be a 
_string | Stripe.Customer | Stripe.DeletedCustomer_.
Thus we force the cast to string if we want to retrieve the customer from stripe using the stripeId (string)

Reply:
`as` is a Typescript keyword, it doesn't really perform real casting as I think you're implying here. Agree with Charles that it feels like a code smell. Are you sure it isn't always returning a customer object and then you can access .id?